### DISCUSSION TITLE
Multiple configs and WagmiProviders

### DISCUSSION THREAD

**Discussion by ntchungse** [wevm/wagmi#3976] [Q&A]
Hi Can I use two config, 1 for wagmi connectors and 1 for web3modal connectors. Wrap two WagmiProviders each others?

Ex:

Configs:
web3Config = defaultWagmiConfig()
wagmiConfig = createConfig

Providers:
 WagmiProvider config={web3Config}
            WagmiProvider config={wagmiConfig}
            WagmiProvider
WagmiProvider

**Comment by nezouse** [comment]
It will always take values from the nearest provider, so no. But if it's possible in your code you can manually pass the config to hooks if that happens only in a few places.

**Reply by ntchungse** [reply]
Thanks for ur help. I am facing an issues that two configs conflict with storing data connections in wagmi.store. How can I change store data of each provider. I tried set createStorage with different key but It seem not working.
